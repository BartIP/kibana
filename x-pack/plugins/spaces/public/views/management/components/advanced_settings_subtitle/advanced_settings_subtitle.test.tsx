/*
 * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 * or more contributor license agreements. Licensed under the Elastic License;
 * you may not use this file except in compliance with the Elastic License.
 */
<<<<<<< HEAD
import { shallow } from 'enzyme';
import React from 'react';
=======
import React from 'react';
import { shallowWithIntl } from 'test_utils/enzyme_helpers';
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
import { AdvancedSettingsSubtitle } from './advanced_settings_subtitle';

describe('AdvancedSettingsSubtitle', () => {
  it('renders as expected', () => {
    const space = {
      id: 'my-space',
      name: 'My Space',
    };
<<<<<<< HEAD
    expect(shallow(<AdvancedSettingsSubtitle space={space} />)).toMatchSnapshot();
=======
    expect(shallowWithIntl(<AdvancedSettingsSubtitle space={space} />)).toMatchSnapshot();
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
  });
});
