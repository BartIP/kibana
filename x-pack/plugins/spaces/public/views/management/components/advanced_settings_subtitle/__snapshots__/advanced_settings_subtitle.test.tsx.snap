// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvancedSettingsSubtitle renders as expected 1`] = `
<React.Fragment>
  <EuiSpacer
    size="m"
  />
  <EuiCallOut
    color="primary"
    iconType="spacesApp"
    size="m"
    title={
      <p>
<<<<<<< HEAD
        The settings on this page apply to the 
        <strong>
          My Space
        </strong>
         space, unless otherwise specified.
=======
        <FormattedMessage
          defaultMessage="The settings on this page apply to the {spaceName} space, unless otherwise specified."
          id="xpack.spaces.management.advancedSettingsSubtitle.applyingSettingsOnPageToSpaceDescription"
          values={
            Object {
              "spaceName": <strong>
                My Space
              </strong>,
            }
          }
        />
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
      </p>
    }
  />
</React.Fragment>
`;
